test/mock-api/responses.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
     {
       _id: '1',
       ladokRoundIds: ['9'],
```

Comment:
ladokRoundIds must be removed from the code at all

Reply:
Done